templates/ts-apollo-fullstack/README.md

```diff
@@ -32,6 +32,9 @@ yarn start:server
 ```
 
 - Generate React components
```

Comment:
Not very accurate description as to be that sounds like it will generate React TSX components.